src/libsync/vfs/off/vfs_off.h

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (C) by Christian Kamm <mail@ckamm.de>
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License
+ * for more details.
+ */
+#pragma once
+
+#include <QObject>
+#include <QScopedPointer>
+
+#include "common/plugin.h"
+#include "common/vfs.h"
+
+namespace OCC {
+
+class VfsOff : public Vfs
+{
+    Q_OBJECT
+
+public:
+    VfsOff(QObject *parent = nullptr);
+    ~VfsOff() override;
+
+    Mode mode() const override;
+
+    QString fileSuffix() const override;
+
+    void stop() override;
+    void unregisterFolder() override;
+
+    bool socketApiPinStateActionsShown() const override;
+    bool isHydrating() const override { return false; }
```

Comment:
All other methods are defined in the corresponding `.cpp` file. Why is this one defined in the header?